output/transform/transform_test.go

```diff
@@ -22,10 +22,10 @@ func TestStatements(t *testing.T) {
 
 	q1 := "SELECT 1"
 	q2 := "SELECT * FROM test"
-	fp1 := util.FingerprintQuery(q1)
+	fp1 := util.FingerprintQuery(q1, "none", -1)
```

Comment:
It might be nice to avoid magic strings here, but that's an existing problem so we don't need to do that in this patch.

Reply:
Agree, I'll not do this here for now, but good cleanup to do some time.